### ISSUE TITLE
Terminated False "Error_Packet "partial packet: expecting X bytes, got: Y""

### ISSUE THREAD

**Issue by spidey** [matterhorn-chat/matterhorn#662]
With an unstable internet connection while trying to send a message I got this error back from the client:

```
An unexpected error has occurred! The exception encountered was:
  Terminated False "Error_Packet \"partial packet: expecting 16408 bytes, got: 9751\"" (Error_Packet "partial packet: expecting 16408 bytes, got: 9751")
Please report this error at https://github.com/matterhorn-chat/matterhorn/issues
```

To me it seems like a normal network connection interrupted exception, I'd expect the sending to be retried automatically and, possibly, the input field could keep the message in an inactive state until it is sent or Matterhorn gives up on retrying.

**Comment by jtdaugherty** [comment]
Thanks for the report!

**Comment by jtdaugherty** [comment]
This is now somewhat addressed in `develop` by 11db5db87750b26d1280d832eaf38778416798f7. Unfortunately we can't accept the suggestions above, since in general there is no way to know whether it is safe to retry an asynchronous task since it could make multiple API calls and complete one but not another. I think if #610 is implemented at some point, we can do something similar to what is described above, but in the mean time, all we can do is ignore this exception (rather than reporting it to the user) on the assumption that the websocket will soon get disconnected and cause the application to enter into a disconnected state.
